app/controllers/food_trucks_controller.rb

```diff
@@ -13,6 +22,7 @@ def new
   end
 
   def create
+    food_truck_params['name'].downcase!
```

Comment:
Our search function is case sensitive, so storing the names in all lowercase allows users to search without being case sensitive because we downcase the search query.

It should still look okay on the site, because the font we chose is all caps anyways